toolchain/sem_ir/dump.cpp

```diff
@@ -211,8 +198,9 @@ LLVM_DUMP_METHOD auto Dump(const File& file, NameScopeId name_scope_id)
             llvm::errs() << "private ";
             break;
         }
-        DumpNoNewline(file, entry.result.target_inst_id());
-        llvm::errs() << "\n";
+        llvm::errs() << StreamedDumpNoNewline(file,
+                                              entry.result.target_inst_id())
```

Comment:
This one seems unfortunately marginally worse than before but others do seem nicer.